db/schema.rb

```diff
@@ -964,7 +964,6 @@
     t.integer "maturity_level"
     t.datetime "date_published"
     t.datetime "practice_pages_updated"
-    t.string "highlight_title"
```

Comment:
I don't think we want to remove this until the feature branch is in master

Reply:
OK... I added it back in.  fyi - I was working on a branch created from master... but since I reviewed the pr that had this change to remove it.. the home page was broke.. since it still had code referencing it.. in the master branch..